uint8 team
uint8 funktion
bool human
geometry_msgs/Pose startPosition
geometry_msgs/Point zielTor
geometry_msgs/Point eigenesTor
